pkgs/development/tools/minizinc/ide.nix

```diff
@@ -2,13 +2,13 @@
 
 mkDerivation rec {
   pname = "minizinc-ide";
-  version = "2.5.5";
+  version = "2.7.6";
 
   src = fetchFromGitHub {
     owner = "MiniZinc";
     repo = "MiniZincIDE";
     rev = version;
-    sha256 = "sha256-0U3KFRDam8psbCaEOcrwqzICAy1oBgo8SFEiR/PMqZk=";
+    sha256 = "sha256-NUGQmOkdnEZq+wr/VhjSR/IBwF565U4QlTkmpjOYJes=";
```

Comment:
Applied against master and building, I get this error (after enabling NIXPKGS_ALLOW_INSECURE due to QtWebKit):

```
source> removing `.git'...
error: hash mismatch in fixed-output derivation '/nix/store/fv24ifwcsw9cc3gawr4wfzanlhbi8v7f-source.drv':
         specified: sha256-NUGQmOkdnEZq+wr/VhjSR/IBwF565U4QlTkmpjOYJes=
            got:    sha256-Fwt4zuIKwurCx0EriwSOea4zAJ7Bmr8Stt0dQ8Ux0cs=
error: 1 dependencies of derivation '/nix/store/z1wqw3gy3d6cfqs80lzv0pjqsdw8njqz-minizinc-ide-2.7.6.drv' failed to build
```

Reply:
Should be fixed